diktat-gradle-plugin/src/functionalTest/kotlin/org/cqfn/diktat/plugin/gradle/Utils.kt

```diff
@@ -0,0 +1,70 @@
+package org.cqfn.diktat.plugin.gradle
+
+import org.gradle.buildinit.plugins.internal.modifiers.BuildInitDsl
+import org.gradle.internal.impldep.org.junit.rules.TemporaryFolder
+import org.gradle.testkit.runner.GradleRunner
+import java.io.File
+import java.util.concurrent.atomic.AtomicInteger
+
+internal val testsCounter = AtomicInteger(0)
+
+internal fun createExampleProject(testProjectDir: TemporaryFolder,
+                                  exampleProject: File,
+                                  buildInitDsl: BuildInitDsl
+) {
+    exampleProject.copyRecursively(testProjectDir.root)
+    val buildFileName = buildInitDsl.fileNameFor("build")
+    File(testProjectDir.root, buildFileName).delete()
+    testProjectDir.newFile(buildFileName).apply {
+        writeText(
+            """
+                plugins {
+                    id("org.cqfn.diktat.diktat-gradle-plugin")
+                }
+                
+                repositories {
+                    mavenLocal()
+                    mavenCentral()
+                }
+            """.trimIndent()
+        )
+    }
+}
+
+/**
+ * @param arguments additional arguments to pass to [GradleRunner]
+ */
+internal fun runDiktat(testProjectDir: TemporaryFolder,
+                      shouldSucceed: Boolean = true,
+                      arguments: List<String> = emptyList(),
+                      configureRunner: GradleRunner.() -> GradleRunner = { this }
+) = GradleRunner.create()
```

Comment:
Can this function be a util method of GradleRunner?

Reply:
Do you mean extension method?

Reply:
Yes

Reply:
I don't see much sense in it: I don't pass `GradleRunner` as a parameter so it won't reduce arguments list; the method starts with call of static method from Java, so it can't be used in an extension.

Reply:
Yes, I was wrong - misunderstood